index.html

```diff
@@ -61,9 +65,13 @@ <h3 class="login-message">Enter your name to join Atari Space Shooter</h3>
             </div>
             <div class="scoreboard" id="scoreBoard"></div>
         </div>
+        <div class="btn">
+            <a href="#" id="fullscreenButton" style="width:50px !important" onclick="goFullScreen();">Full screen</a>
```

Comment:
Using `!important` in CSS is considered an anti-pattern. See https://stackoverflow.com/questions/8360190/is-it-bad-to-use-important-in-css-property.  Please find a different way to override the property without writing 'hard-to-maintain' code.